coverage/.resultset.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
you need to add the coverage folder to the `.gitignore`

Reply:
http://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/

Reply:
Added to .gitignore and successfully removed folder from repository.